security_monkey/datastore.py

```diff
@@ -62,16 +70,37 @@ class Account(db.Model):
     id = Column(Integer, primary_key=True)
     active = Column(Boolean())
     third_party = Column(Boolean())
-    name = Column(String(32))
+    name = Column(String(32), unique=True)
+    s3_name = Column(String(64)) # (deprecated-custom)
+    number = Column(String(12))  # (deprecated-identifier) Not stored as INT because of potential leading-zeros.
     notes = Column(String(256))
-    s3_name = Column(String(64))
-    number = Column(String(12))  # Not stored as INT because of potential leading-zeros.
+    identifier = Column(String(256))  # Unique id of the account, the number for AWS.
     items = relationship("Item", backref="account", cascade="all, delete, delete-orphan")
     issue_categories = relationship("AuditorSettings", backref="account")
-    role_name = Column(String(256))
+    role_name = Column(String(256)) # (deprecated-custom)
```

Comment:
If you'd like to keep `account.identifier` and `account.number`, I would think you'd need to update `security_monkey/account_manager.py`::`_populate_account(...)` so that it sets `account.number` as well as `account.identifier` (~ line 128)

Otherwise old and new accounts will be inconsistent.

    def _populate_account(self, account, account_type_id, name, active, third_party,
                          notes, identifier, custom_fields=None):
        """
        Creates account DB object to be stored in the DB by create or update.
        May be overridden to store additional data
        """
        account.name = name
        account.identifier = identifier
        account.notes = notes


What would the timeline be for removing the deprecated fields?